clients/mobile/src/userWorkflow/Profile.tsx

```diff
@@ -1,21 +1,73 @@
 /* eslint-disable */
 // This page is not implemented yet just demo.
 import React from 'react'
-import { Text, Button, View, StyleSheet } from 'react-native'
+import { Text, Button, View, StyleSheet, TouchableOpacity, Image, FlatList, ScrollView, Platform } from 'react-native'
 import { signOut } from 'aws-amplify/auth'
 import { useTheme } from '../setup/theme/hooks/useTheme'
 import { ProfileScreenNavigationProp } from '../setup/navigation/navigationTypes'
 import { SCREENS } from '../setup/constant/screens'
 import { Cache } from 'aws-amplify/utils';
 import AsyncStorage from '@react-native-async-storage/async-storage';
-import { Platform } from 'react-native';
 import { useAuth } from '../authentication/context/useAuth'
+import { Ionicons } from '@expo/vector-icons'
 
 interface ProfileScreenProps {
   navigation: ProfileScreenNavigationProp;
 }
 
-export default function Profile({ navigation }: ProfileScreenProps) {
+interface Badge {
+  id: number;
+  icon: string;
+  label: string;
+}
+
+interface Achievement {
+  id: number;
+  icon: string;
+  label: string;
+  subLabel: string;
+}
+
+interface Post {
+  id: number;
+  title: string;
+  donationPoint: string;
+  dateTime: string;
+  urgent: boolean;
+}
+
+const Profile: React.FC<ProfileScreenProps> = ({ navigation }) => {
+// dummy data
+const badges: Badge[] = [
+  // { id: 1, icon: 'badge', label: 'Gold' },
```

Comment:
?

Reply:
This page was in progress and some dummy data was there to test things as the API wasn't ready then. Removing the comments.